README.md

```diff
@@ -582,7 +582,30 @@ Vegetation Phenology and Productivity Parameters is available in Sentinel Hub, r
 - [RSLOPE visualisation script](copernicus_services/vpp-slope-of-senescent-period-rslope)
 - [LSLOPE visualisation script](copernicus_services/vpp-slope-of-greening-up-period-lslope)
 - [SPROD visualisation script](copernicus_services/vpp-seasonal-productivity-sprod)
-- [TPROD visualisation script](copernicus_services/vpp-total-productivity-tprod) 
+- [TPROD visualisation script](copernicus_services/vpp-total-productivity-tprod)
+
+### High Resolution Snow & Ice Monitoring
+
+The High-Resolution Snow & Ice Monitoring service ([HR-S&I](https://land.copernicus.eu/pan-european/biophysical-parameters/high-resolution-snow-and-ice-monitoring)) is part of the Copernicus Land Monitoring Service (CLMS). The snow aspect of the service provides products measuring Snow cover (FSC, FSTOC, FSCOG, GFSC), Snow state conditions (WDS, SWS) and persistent snow area (PSA). There are aso ice products that measure ice cover (RLIE) and aggregated river and lake ice extent (ARLIE).

```

Comment:
For a chance to describe each collection, I suggest to have an entry for each collection/product separately and list the associated scripts underneath.

Reply:
Sure, will split them into the seperate collections so there isn't one very long list!